students/ShujinCao/lesson04/assignment/test_basic_operations.py

```diff
@@ -0,0 +1,90 @@
+"""
+tests
+"""
+import pytest
+from basic_operations import *
+from customer_model import Customer
+
+good_customer2 = {
+    'customer_id': '2WDIll8IDO',
+    'name': 'Suzie2',
+    'last_name': 'Edgar',
+    'home_address': '122 Brown St., Seattle, WA 98121',
+    'phone_number': '122-333-444',
+    'email_address': '2suzie@gmail.com',
+    'status': 'Active',
+    'credit_limit': 40.00}
+
+good_customer = {
+    'customer_id': 'WDIll8IDO',
+    'name': 'Suzie',
+    'last_name': 'Edgar',
+    'home_address': '122 Brown St., Seattle, WA 98121',
+    'phone_number': '123-333-444',
+    'email_address': 'suzie@gmail.com',
+    'status': 'Active',
+    'credit_limit': 40.00}
+
+bad_customer = {'customer_id':'8675309',
+                'name': 'Jenny',
+                'last_name': 'Igotyonumber',
+                'home_address': 'Beverly Hills Plaza, Compton, Ca. 90210',
+                'phone_number': '198-765-4321',
+                'email_address': 'Jenny_Igotyonumber@gmail.com',
+                'status': 'Active',
+                'credit_limit': 40.00}
+
```

Comment:
It looks like you are not creating/deleting a database for tests. This is really a must. It is really important to understand how to separate testing from production. Easiest if you use fixtures.